tests/php/test-class-plugin-base.php

```diff
@@ -44,6 +44,16 @@ public function test_locate_plugin() {
 	}
 
 	/**
+	 * Test relative_path.
+	 *
+	 * @see Plugin_Base::relative_path()
+	 */
+	public function test_relative_path() {
+		$this->assertEquals( 'plugins/foo-bar', $this->plugin->relative_path( '/srv/www/wordpress-develop/src/wp-content/plugins/foo-bar', 'wp-content', '/' ) );
```

Comment:
The `'/srv/www/wordpress-develop/src` path wouldn't work in a non-VVV context or if the unit tests aren't being run in `wordpress-develop`.